submit-web/src/styles/theme.tsx

```diff
@@ -1,6 +1,16 @@
 import "@/styles/App.scss";
-import { createAppTheme } from 'epic.theme';
+import { createAppTheme } from "epic.theme";
 
 // Any theme overrides should be passed into the createAppTheme.
 
-export const theme = createAppTheme({});
+export const theme = createAppTheme({
+  components: {
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          height: "40px",
```

Comment:
I'm worried about this, this has caused problems in Track where text has been squished and sometimes it overflowed especially when the text wraps. Maybe also then not allow text to wrap in buttons (some css property)

Reply:
how should we set this up then?

Reply:
please look into it or remove this and do it in another PR